dashboard/package.json

```diff
@@ -20,12 +20,18 @@
   "bugs": {
     "url": "https://github.com/ucberkeley/sis-qa-test-auto/issues"
   },
-  "devDependencies": {
+  "dependencies": {
+    "browserify": "~13.0.0",
```

Comment:
@mDibyo Sounds good, thanks for clarifying. We just need to make sure we keep that line of thought moving forward.